app/controllers/BackfillChunksAdapter.java

```diff
@@ -15,6 +19,8 @@
  */
 class BackfillChunksAdapter implements BackfillCallback {
 
+    private final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Do we want to use BridgeObjectMapper.get()? This way, we don't have to create an ObjectMapper instance for every class.

Minor: If we still want to use ObjectMapper directly, we should declare it static. That said, since BackfillChunksAdapter is singleton, it's not a drastic difference.